build/regressions.config.yml

```diff
@@ -109,6 +109,18 @@ sources:
         - sebastienros
       template: reliability
       healthCheck: true
+  - name: Native AOT Reliability
+    table: NativeAotBenchmarks
```

Comment:
In the `PublishedSize` regression above, I guess it is using `table: TrendBenchmarks`. But this one uses `NativeAotBenchmarks`.

Should I have used a different table above?